src/main/java/fi/aalto/cs/apluscourses/ui/base/DialogBaseHelper.java

```diff
@@ -0,0 +1,46 @@
+package fi.aalto.cs.apluscourses.ui.base;
+
+import java.awt.event.KeyEvent;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import javax.swing.JButton;
+import javax.swing.JComponent;
+import javax.swing.JDialog;
+import javax.swing.JPanel;
+import javax.swing.KeyStroke;
+
+/**
+ * The contents of this class are fully auto-generated by the IntelliJ IDEA.
+ */
+public class DialogBaseHelper extends JDialog {
```

Comment:
I assume there's no point in fixing sonar's complaints here if this is auto-generated?

Reply:
not really